src/components/home/Home.tsx

```diff
@@ -54,6 +55,9 @@ const LoadingPetCards = () => {
           <LoadPlaceHolder />
           <LoadPlaceHolder />
         </PetCardFlex>{" "}
+        <Button color="primary">
+          <img className="rotate" src={load} alt="" />
```

Comment:
can you something to the alt tag maybe like "spinner"

Reply:
Done